code/game/objects/structures/crates_lockers/closets/wardrobe.dm

```diff
@@ -189,12 +189,6 @@
 	new /obj/item/clothing/under/color/jumpskirt/lightpurple(src)
 	new /obj/item/clothing/under/color/green(src)
 	new /obj/item/clothing/under/color/jumpskirt/green(src)
-	new /obj/item/clothing/mask/bandana/red(src)
-	new /obj/item/clothing/mask/bandana/red(src)
-	new /obj/item/clothing/mask/bandana/blue(src)
-	new /obj/item/clothing/mask/bandana/blue(src)
-	new /obj/item/clothing/mask/bandana/gold(src)
-	new /obj/item/clothing/mask/bandana/gold(src)
```

Comment:
what's up with this?

Reply:
mixed walldrobes (wosets?) were spawning with too much THINGS in them, and would not close with everything in them. so I removed all of the bandanas, there's probably a better solution to this.